scijava-ops-image/src/main/java/module-info.java

```diff
@@ -27,9 +27,19 @@
  * #L%
  */
 
+// HACK: Open the scijava-ops-image module completely.
+//
+// What's supposed to be needed is:
+//   opens org.scijava.ops.tutorial to org.scijava.ops.engine;
+//
+// Unfortunately, doing that results in the following error at test time:
+//
+//   Error occurred during initialization of boot layer
+//   java.lang.module.FindException: Error reading module:
+//   .../scijava/scijava-ops-image/target/classes
```

Comment:
Hmm, it might be good to add the HACK comment to the scijava-ops-tutorial `module-info.java`, and to replace `tutorial`->`image` here.

Reply:
Additionally, I believe that the module needs to be open to scijava-ops-engine anyways in production, so it's not a total hack...

Might this be fixed by adding `opens org.scijava.ops.image(.foo.bar...) to org.scijava.ops.engine` and then adding some opens via `scijava.surefire.args`?

Reply:
I tried adding the 105 opens statements to open every scijava-ops-image package containing an Op to scijava-ops-engine:
```java
opens org.scijava.ops.image to org.scijava.ops.engine;
opens org.scijava.ops.image.adapt to org.scijava.ops.engine;
... 101 more lines ...
opens org.scijava.ops.image.types.maxValue to org.scijava.ops.engine;
opens org.scijava.ops.image.types.minValue to org.scijava.ops.engine;
```
and it almost works, but the `OpRegressionTest` still fails, along with a couple of others.

So I give up for now! 925bfaef9c492977f8110597e796432e551acbca